source/adios2/toolkit/transport/file/FilePointer.cpp

```diff
@@ -48,16 +48,16 @@ void FilePointer::Open(const std::string &name, const OpenMode openMode)
 
     if (m_OpenMode == OpenMode::Write)
     {
-        m_File = std::fopen(name.c_str(), "w");
+        m_File = fopen(name.c_str(), "wb");
```

Comment:
FYI. The only thing that worked was removing the warnings explicitly with pragmas:

#ifdef _WIN32
#pragma warning(disable : 4996) // fopen
#endif